osu.Game/Screens/Edit/Components/Timelines/Summary/Parts/MarkerPart.cs

```diff
@@ -78,34 +83,113 @@ public MarkerVisualisation()
                 RelativePositionAxes = Axes.X;
                 RelativeSizeAxes = Axes.Y;
                 AutoSizeAxes = Axes.X;
+                Masking = true;
                 InternalChildren = new Drawable[]
                 {
-                    new Triangle
-                    {
-                        Anchor = Anchor.TopCentre,
-                        Origin = Anchor.BottomCentre,
-                        Scale = new Vector2(1, -1),
-                        Size = new Vector2(10, 5),
-                    },
-                    new Triangle
-                    {
-                        Anchor = Anchor.BottomCentre,
-                        Origin = Anchor.BottomCentre,
-                        Size = new Vector2(10, 5),
-                    },
                     new Box
                     {
                         Anchor = Anchor.Centre,
                         Origin = Anchor.Centre,
                         RelativeSizeAxes = Axes.Y,
                         Width = 1.4f,
                         EdgeSmoothness = new Vector2(1, 0)
+                    },
+                    new VerticalTriangles
+                    {
+                        Anchor = Anchor.Centre,
+                        Origin = Anchor.Centre,
+                        RelativeSizeAxes = Axes.Y,
+                        Width = 10
                     }
                 };
             }
 
             [BackgroundDependencyLoader]
             private void load(OverlayColourProvider colours) => Colour = colours.Highlight1;
+
+            private partial class VerticalTriangles : Sprite
```

Comment:
Would add xmldoc or comments somewhere around here to mention that we're using a custom implementation to get matching antialiasing on the triangles.